README.md

```diff
@@ -6,11 +6,10 @@
 
 #### Requirements ####
 
-You will need the following things installed:
- - [NodeJs](https://nodejs.org/en/ "NodeJs")
- - [Go 1.6](https://golang.org/ "Go")
- 
+Requires [Go 1.6](https://golang.org/) to compile and run, along with something
```

Comment:
Generally, but afaik the thing only *requires* 1.6. Unless I've missed something